tvb_build/docker/Dockerfile-win

```diff
@@ -27,6 +26,7 @@ RUN activate tvb-run && python setup.py develop
 
 COPY requirements_group requirements.txt
 RUN activate tvb-run && pip install -r requirements.txt
+RUN pip install -U scipy
```

Comment:
do we still need the pip install -U scipy if we removed the allensdk from req file ?

Reply:
I don't think so. I will remove it and build it again.